src/actions/guides/database/deleting_records.cr

```diff
@@ -0,0 +1,169 @@
+class Guides::Database::DeletingRecords < GuideAction
+  guide_route "/database/deleting-records"
+
+  def self.title
+    "Deleting records"
+  end
+
+  def markdown : String
+    <<-MD
+    ## Deleting Records
+
+    ### Delete one
+
+    Deleting a single record is actually done on the [model](#{Guides::Database::Models.path}) directly.
+    Since each query returns an instance of the model, you can just call `delete` on that record.
```

Comment:
This isn't quite right since queries can return a Queryable and not a collection. I think it may be more clear to leave this off and show only the example. I think it is pretty clear like that